testing-commandments.md

```diff
@@ -0,0 +1,22 @@
+# HolidayExtras Automated Testing Commandments
```

Comment:
Commandments is a fun analogy but there are always exceptions to rules and there may be occasions where the below can be broken. I would submit that such cases should be throughly considered and reviewed.

My point is juniors may take "commandments" too literarily and maybe we should have a disclaimer.